Lib/test/test_zipfile.py

```diff
@@ -8,6 +8,8 @@
 import struct
 import zipfile
 import unittest
+import tempfile
+import shutil
```

Comment:
It looks to me like the imports in this file started as being sorted alphabetically.  `unittest` breaks this, but I would suggest resorting these imports.